socorro/processor/processor_app.py

```diff
@@ -131,6 +142,45 @@ def _transform(self, crash_id):
                 crash_id
             )
             self.config.logger.info('saved - %s', crash_id)
+        except Exception as exception:
+            # Immediately capture this as local variables.
+            # During this error handling we're going to be using other
+            # try:except: constructs (e.g. swallowing raven send errors)
+            # so we can't reference `sys.exc_info()` later.
+            exc_type, exc_value, exc_tb = sys.exc_info()
+
+            if self.config.sentry and self.config.sentry.dsn:
+                try:
+                    if isinstance(exception, PolyStorageError):
```

Comment:
Hmm... But you make a good point. If we use `collections.Sequence` instead, we might not need that explicit import from the `crashstorage_base.py`. That'd be nice.